src/nn_maze_solver.py

```diff
@@ -0,0 +1,113 @@
+import os
+import random
+
+import numpy as np
+import tensorflow as tf
+
+from src.maze import Maze
+from src.mouse import Mouse
+
+
+class Solver:
+    def __init__(self, map_file_path: str):
+        os.environ["TF_CPP_MIN_LOG_LEVEL"] = "99"
+        tf.get_logger().setLevel("ERROR")
+        tf.autograph.set_verbosity(0)
+        tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+        tf.keras.utils.disable_interactive_logging()
```

Comment:
```suggestion
        tf.keras.utils.disable_interactive_logging()
```
This one will be enough for us, sry because it was my bad code.